src/main/java/ca/corefacility/bioinformatics/irida/service/impl/export/NcbiExportSubmissionServiceImpl.java

```diff
@@ -102,7 +103,13 @@ public NcbiExportSubmission create(NcbiExportSubmission object) throws Constrain
 			throws IllegalArgumentException {
 		return super.list(page, size, order, sortProperties);
 	}
-	
+
```

Comment:
Add method comment. Probably just @inheritDoc

Reply:
Updated in 4af6a7e